.gitignore

```diff
@@ -10,3 +10,4 @@ bookmarks
 network-security.data
 quelpa/
 modules/.dir-locals.el
+.aider*
```

Comment:
what is this?

Reply:
Session files created by [aider](https://aider.chat/)